src/components/dialogs/parameters/parameters.js

```diff
@@ -732,9 +800,11 @@ const Parameters = ({ user, isParametersOpen, hideParameters }) => {
                                         />
                                     )}
                                 </TabPanel>
-                            )
-                        }
-                        {
+                            )}
+                        {!isUnavailableService(
+                            unavailableOptionalServices,
+                            TAB_VALUES.dynamicSimulationParamsTabValue
+                        ) &&
```

Comment:
those few lines are repeated a lot, we could create a hook with this behaviour : 
- takes an optional service name as a param
- useSelector on the optionalServicesNames in redux
- returns a boolean "isUnavailable"